src/main/java/ua/bookstore/online/controller/BookController.java

```diff
@@ -65,6 +72,7 @@ public BookDto updateBook(
         return bookService.update(id, bookDto);
     }
 
+    @PreAuthorize("hasRole('USER')")
```

Comment:
```suggestion
    @PreAuthorize("hasRole('USER')")
```
Is it correct that only the User can enter these endpoints (getBookById,getAll(),searchBooks())?

Reply:
Admin and manager include the role 'user'.